app/models/system_console.rb

```diff
@@ -53,10 +53,10 @@ def self.launch_proxy(remote_address, remote_port)
       return nil
     end
 
-    command = '/usr/bin/socat', "TCP-LISTEN:#{local_port},fork", "TCP:#{remote_address}:#{remote_port}"
+    command = ["/usr/bin/socat", "TCP-LISTEN:#{local_port},fork", "TCP:#{remote_address}:#{remote_port}"]
```

Comment:
Also, would a simple to_i fix the warnings?

```suggestion
    command = ["/usr/bin/socat", "TCP-LISTEN:#{local_port.to_i},fork", "TCP:#{remote_address.to_i}:#{remote_port.to_i}"]
```

Think a `to_i` will make commands not come through.
Also think that brakeman is smart enough to know that.

Reply:
Don't think you can do that with remote_address

Reply:
:+1: yes.
darn - not sure if we can make that safe w/o going through too much work.
We are pretty sure that this will not be bad stuff

Reply:
AwesomeSpawn handles all of those, so I don't see the need for it.